libs/java/auth_core/src/main/java/com/yahoo/athenz/auth/util/PrivateKeyStoreUtil.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright The Athenz Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yahoo.athenz.auth.util;
+
+import com.yahoo.athenz.auth.ServerPrivateKey;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+import java.security.PrivateKey;
+import java.util.function.Function;
+
+public class PrivateKeyStoreUtil {
+    private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+    private static final String ZMS_SERVICE = "zms";
+    private static final String ZTS_SERVICE = "zts";
+    private static final String MSD_SERVICE = "msd";
+
+    private static final String ATHENZ_PROP_ZMS_KEY_NAME    = "athenz.aws.zms.key_name";
```

Comment:
For the key names, we should make them cloud specific. It's not right to say I have a keystore implementation for gcp but my property name is based on aws. So it should be:

"athenz." + cloud + "." + service + ".key_name"

add add the cloud as an argument to the getPrivateKeyFromCloudParameter method

Reply:
Good point. Updated the code.